Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -19,7 +19,6 @@ file that was distributed with this source code.
     <select id="{{ id }}_autocomplete_input_v4" data-sonata-select2="false"
         {%- if read_only is defined and read_only %} readonly="readonly"{% endif -%}
         {%- if disabled %} disabled="disabled"{% endif -%}
-        {%- if required %} required="required"{% endif %}
```

Comment:
I'm not sure they are even supposed to be there anymore : https://select2.github.io/announcements-4.0.html#hidden-input